darwin/importer/importer.py

```diff
@@ -264,8 +265,30 @@ def import_annotations(
         remote_classes = build_main_annotations_lookup_table(team_classes)
 
     # Need to re parse the files since we didn't save the annotations in memory
-    for local_path in set(local_file.path for local_file in local_files):
+    from darwin.dataset.utils import exhaust_generator
+
+    result = exhaust_generator(
+        upload_generator(
+            local_files,
+            importer,
+            local_files_missing_remotely,
+            remote_files,
+            remote_classes,
+            dataset.client,
+            attributes,
+            append,
+        ),
+        len(filenames),
+        True,
+    )
+    if len(result) != len(filenames):
+        print(f"error, only {len(result)} files succeeded (expected {len(filenames)})")
```

Comment:
Do we want to raise here?

Reply:
I would say so. We could maybe catch the exception in `cli_functions` to print a nicer error/warning message. I think we shouldn't `print` in this module at all, even though we are doing it multiple times in this function.